packages/client/src/components/add-liquidity/add-liquidity-v3.scss

```diff
@@ -0,0 +1,179 @@
+.add-v3-container {
+    width: 600px;
+    position: relative;
+    background: var(--bgPrimary);
+    padding: 0;
+    border: 1px solid var(--borderDeep);
+    box-shadow: 0 0 5px 1px var(--bgContainer);
+    display: inline-flex;
+    flex-direction: column;
+    // justify-content: space-between;
+    // font-family: 'Barlow Condensed';
+    .form-group {
+        margin: 0;
+    }
+    .token-pair-selector {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        > div {
+            // border: 1px solid yellow;
+            margin: 1rem;
+        }
+    }
+
+    .dropdown {
+        outline: none;
+        .btn-primary {
+            background: var(--bgSecondary);
+            border: 1px solid var(--bgDeep);
+            &:focus {
+                outline: none;
+                box-shadow: none;
+                border: 1px solid var(--borderAccentAlt);
+            }
+        }
+    }
+    .dropdown-menu {
+        // border: 1px solid red;
+        border-radius: 0;
+        background: var(--bgSecondary);
+        .dropdown-item {
+            color: var(--faceSecondary);
+            &.active,
+            &:hover {
+                color: var(--facePrimary);
+                background: var(--bgAccent);
+            }
+        }
+    }
+
+    .token-and-wallet {
+        padding: 1rem;
+        display: flex;
+        justify-content: space-between;
+        width: 100%;
+    }
+
+    .balances-container {
+        // border: 1px solid var(--borderMain);
+        font-size: 0.8rem;
+        color: var(--faceDeep);
+        display: flex;
+        flex-direction: column;
+        position: relative;
+        right: 0;
+    }
+
+    .wallet-fees {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        align-items: stretch;
+    }
+
+    .tab-container {
+        display: flex;
+        .tab {
+            font-size: 1.2rem;
+            text-align: center;
+            padding: 0.5rem 2rem;
+            flex-grow: 1;
+            h3 {
+                font-family: 'Barlow Condensed';
```

Comment:
shouldn't this be a global var

Reply:
yea we currently have 2 different font types that we use without a standard way of what goes where. so until then.